src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php

```diff
@@ -31,7 +31,7 @@ public function process(ContainerBuilder $container)
         $aliases = $container->getAliases();
         $exprLangProviders = $container->getExpressionLanguageProviders();
 
-        foreach ($container->getExtensions() as $extension) {
```

Comment:
Correct, currently it is in order of AppKernel registration, but the change reverses that order (so that by the end of that foreach things are actually in the AppKernel order)